arch/cpu/cc26xx-cc13xx/cc26xx.ld

```diff
@@ -96,6 +96,8 @@ SECTIONS
     /* These symbols are used by the stack check library. */
     _stack = .;
     _stack_origin = ORIGIN(SRAM) + LENGTH(SRAM);
+    _heap = ORIGIN(SRAM);
+    _eheap = ORIGIN(SRAM) + LENGTH(SRAM);
```

Comment:
I'm afraid this opened yet another can of worms.

I disagree with this definition of `_heap`. It should be after the static data, not at the start of the memory.

I don't disagree with setting `_eheap` at the end of the RAM.

I suspect that `_heap` is equal to  `_stack` on all platforms (and all platforms that support the stack checker already define the latter, so the former can be added by `LDFLAGS += -Wl,--defsym=_heap=_stack`). Same for `_eheap` and `_stack_origin`.

Reply:
Hm you are of course right. I'll update.

Reply:
@atiselsts what do you think of

```
    /* These symbols are used by the stack check library. */
    _stack = .;
    _stack_origin = ORIGIN(SRAM) + LENGTH(SRAM);
    _heap = _stack;
    _eheap = _stack_origin;
```